metaflow/cli.py

```diff
@@ -519,10 +591,13 @@ def _check(echo, graph, flow, environment, pylint=True, warnings=False, **kwargs
 
 def print_metaflow_exception(ex):
     echo_always(ex.headline, indent=True, nl=False, bold=True)
+    location = ""
+    if ex.source_file is not None:
+        location += " in file %s" % ex.source_file
     if ex.line_no is None:
-        echo_always(":")
-    else:
-        echo_always(" on line %d:" % ex.line_no, bold=True)
+        location += " on line %d" % ex.line_no
```

Comment:
This part fails as line_no is None but the string is expecting an integer.

Reply:
should be able to hit this with any pypi/conda flow and failing to apply `--environment conda` on the CLI at least.

Reply:
well yes, I am an idiot -- the condition is reversed above. Let me fix that.